oml/functional/metrics.py

```diff
@@ -297,6 +494,38 @@ def _to_tensor(array: Union[np.ndarray, torch.Tensor]) -> torch.Tensor:
         raise TypeError("Wrong type")
 
 
+def _check_if_nonempty_integers_and_positive(seq: Sequence[int], name: str) -> None:
+    """
+    Check whether ``args`` is not empty and all its elements are positive integers.
+
+    Args:
+        seq: A sequence.
+        name: A name of the sequence in case of exception should be raised.
+
+    """
+    if not len(seq) > 0 or not all([isinstance(x, int) and (x > 0) for x in seq]):
+        raise ValueError(f"{name} is expected to be a tuple with positive integers, but got {seq}")
+
+
+def _clip_max_with_warning(arr: Tuple[int, ...], max_el: int) -> Tuple[int, ...]:
```

Comment:
let's move it to `oml.utils.misc` and remove the underscore from the name
what do you think?

Reply:
I don't know whether this function could be useful in some other places, except functional/metrics.py.

Reply:
As for this one -- I agree, it's more specific.